### DISCUSSION TITLE
How exactly are flowbite tables "responsive"?

### DISCUSSION THREAD

**Discussion by maxmarcon** [themesberg/flowbite#101] [Q&A]
Hello. First of all thank you so much for creating this very useful set of simple-to-use component! 🎉 

I have a question. In the description of the flowbite table component: 

https://flowbite.com/docs/components/tables/

The table is described as being "responsive". However, it really isn't, at least not in the way I understand the term "responsive".
A responsive table is typically something like this:

https://tailwindcomponents.com/component/mobile-responsive-table

The table changes the layout when the screen shrinks and reorganizes its rows and headers in a vertical layout. The flowbite table doesn't do that.

So in what sense are flowbite tables "responsive" ?

Thanks!

**Comment by MatthewRorke** [comment]
I am bumping this because it has been well over a year and there has been no response, despite this question being extremely valid.

Can we please get some clarification on how setting an overflow property makes a table responsive? Horizontal scrolling is really not ideal in this scenario, whereas the Tailwind Table here: https://tailwindcss.com/docs/table-layout demonstrates a better level of responsiveness.

**Reply by yassinetrbls** [reply]
Thanks !

**Comment by zoltanszogyenyi** [comment]
Hey @MatthewRorke,

We'll have a look at this for the next release. We're thinking of offering multiple methods of responsive features, one of them still being the horizontal scroll on mobile devices.

I think that one of the issues that arise is when there are too many columns to fit them in on one row on mobile devices (the examples from Tailwind CSS only show 3 columns and it's obvious that there's enough space for all of them to fit in).

One method would be to truncate the longer descriptions and also wrap on multiple lines, but it's also very table data specific as to how we should go about wrapping them on smaller device widths.

However, it is true that we didn't yet explore all of the methods since we used a fallback for horizontal scroll which should cover any amount of data. We'll look into this for sure.

Cheers,
Zoltan
